crates/api_models/src/payouts.rs

```diff
@@ -437,8 +437,6 @@ pub struct PayoutAttemptResponse {
     pub connector_transaction_id: Option<String>,
     /// If the payout was cancelled the reason provided here
     pub cancellation_reason: Option<String>,
-    /// Provide a reference to a stored payout method
-    pub payout_token: Option<String>,
```

Comment:
why is this removed? Is this not being used

Reply:
Yes, this is a temporary token which is not necessary for analytics